dojo/templates/dojo/view_finding.html

```diff
@@ -473,48 +473,64 @@ <h4>Similar Findings<span class="pull-right"><a data-toggle="collapse" href="#si
                                     {% with similar.notes.count as note_count %}
                                         ({{ note_count }} note{{ note_count|pluralize }})
                                     {% endwith %}
                                 </td>
-                                <td>{{ similar.status }}{% if similar.duplicate_finding_id %}, <a href="{% url 'view_finding' similar.duplicate_finding_id %}">Original</a> {% endif %}</td>
+                                <td><a href="{% url 'view_test' similar.test_id %}">{{ similar.test.test_type.name }}</a></td>
+                                <td><a href="{% url 'view_engagement' similar.test.engagement_id %}">{{ similar.test.engagement.name }}</a>
+                                {% if similar.test.engagement.version%}
+                                    <sup>
+                                          <a target="_blank" class="tag-label tag-version" data-toggle="tooltip" data-placement="bottom" title="Product Version: {{ eng.version }}">
+                                          {{ similar.test.engagement.version }}</a>
+                                    </sup>
+                                {% endif %}
+                                </td>
+                                {% if finding.cwe > 0 %}
                                 <td>
-                                    {% if similar.tags %}
-                                        <small>
-                                            {% for tag in similar.tags %}
-                                                <a title="Search {{ tag }}" class="tag-label tag-color"
-                                                   href="{% url 'simple_search' %}?query={{ tag }}">{{ tag }}</a>
-                                            {% endfor %}
-                                        </small>
-                                    {% endif %}
+                                    <a target="_blank" href="https://cwe.mitre.org/data/definitions/{{ finding.cwe }}.html">
+                                        <i class="fa fa-external-link"></i> {{ finding.cwe }}
+                                    </a>
                                 </td>
-                                <td><a href="{% url 'view_test' similar.test_id %}">{{ similar.test.test_type.name }}</a></td>
-                                <td><a href="{% url 'view_engagement' similar.test.engagement_id %}">{{ similar.test.engagement.name }}</a></td>
-                                <td>{{ similar.test.engagement.version }}</td>
+                                {% endif %}
                                 <td>{{ similar.cve }}</td>
-                                <td>{{ similar.file_path }}{% if similar.line > 0 %} (Line {{ similar.line }}){% endif %}</td>
+                                <td title="{{similar.file_path}}">{{ similar.file_path|truncatechars_html:20 }}{% if similar.line > 0 %} (Line {{ similar.line }}){% endif %}</td>
```

Comment:
I would still want to be able to copy the full file path here. The hover alone does not allow me to do that.

Reply:
hmm ok, let me see. the problem with these long file paths is that they don't wrap very well.

Reply:
I have another tool that resolved this by putting a small "copy" button on the left of the string. Would that be possible maybe? Therefore, no need to display the whole line.. we can just copy it and see it elsewhere in full.

Reply:
was thinking about that as well, just need to find out what the proper way to copy to clipboard is in 2020.

Reply:
ok, used a library make it "just work"